Support non-unique meta key in autosave (#61)
adamsilverstein/wp-post-meta-revisions: readme.txt

Further development of the code for this plugin will continue on its <a href="https://github.com/adamsilverstein/wp-post-meta-revisions">GitHub repository</a>. Pull requests welcome!

> To use this plugin, you must be running WordPress 4.9.8 or newer, a function were added in 4.9.8 that are required for this implementation.

Author: ```suggestion
To use this plugin, you must be running WordPress 4.9.8 or newer, a function was added in 4.9.8 that is required for this implementation.
```
